CONTRIBUTING.md

```diff
@@ -1,4 +1,83 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
 # Contributing
 
-For detailed contribution guideline, please check the [contribution guideline of Apache Ozone repository](https://github.com/apache/ozone/blob/master/CONTRIBUTING.md).
+For general contribution guideline, please check the [Apache Ozone repository](https://github.com/apache/ozone/blob/master/CONTRIBUTING.md).
+
+Development of the `ozone-runner` image happens on branch `master`, relevant changes are cherry-picked to branch `jdk11`.
+
+## Local Build and Test
+
+### Building
+
+The image can be built simply by running the helper script `build.sh`:
+
+```bash
+$ ./build.sh
+...
+ => => naming to docker.io/apache/ozone-runner:dev
+```
+
+### Testing
+
+To try the image locally with Ozone acceptance tests, define the version to be used:
+
+```bash
+export OZONE_RUNNER_VERSION=dev
+```
+
+then run [acceptance tests](https://github.com/apache/ozone/blob/master/hadoop-ozone/dist/src/main/compose/README.md) as needed.
+
+## GitHub Workflows
+
+If this is your first time working on the image, please enable GitHub Actions workflows after forking the repo.
+
+### Building
+
+CI checks are triggered by pushing changes to your fork.
+
+GitHub Actions workflow runs in your fork are listed at `https://github.com/<username>/ozone-docker-runner/actions`.
+
+The image is built and tagged with commit SHA in GitHub, ready for testing integration with other repos.
+
+Images built in your fork are available at `https://github.com/<username>/ozone-docker-runner/pkgs/container/ozone-runner`.
+
+These images can be shared with other developers for feedback.
```

Comment:
nit. we can collapse this into one paragraph. I don't think we need one sentence per line.